templates/logging.template

```diff
@@ -23,10 +23,7 @@ Parameters:
     Type: String
     Default: true
 Conditions:
-  IsGovCloud:
-    !Equals
-    - us-gov-west-1
-    - !Ref AWS::Region
+  IsGovCloud: !Or [!Equals [!Ref "AWS::Region", us-gov-west-1], !Equals [!Ref "AWS::Region", us-gov-east-1]]
```

Comment:
```suggestion
  IsGovCloud: !Equals [!Ref "AWS::Partition", aws-us-gov]
```

recommend using the [`AWS::Partition` pseudoparameter](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/pseudo-parameter-reference.html#cfn-pseudo-param-partition) instead